.kiro/steering/spec-kickoff.md

```
@@ -0,0 +1,386 @@
+# Spec Kickoff Workflow
+
+When user says "kickoff {spec-name}", execute this complete workflow to set up a spec for autonomous development.
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)